test/sagemaker_tests/mxnet/inference/integration/sagemaker/test_neuron_hosting.py

```diff
@@ -44,6 +44,7 @@ def test_neuron_hosting(sagemaker_session, ecr_image, instance_type, framework_v
         SCRIPT_PATH,
         framework_version=framework_version,
         image_uri=ecr_image,
+        model_server_workers=1,
```

Comment:
Why is this change necessary? Could you please add a comment here (if it is just a comment, CI tests aren't necessary) to explain why the number of workers needs to be hardcoded to 1?

If it is something that accompanies the upgrade to SDK 1.17.0, then please mention this in the PR Description for posterity.